Makefile.docs

```diff
@@ -1,5 +1,5 @@
-userdocs/src/usage/schema.json:
-	cp ./pkg/apis/eksctl.io/v1alpha5/assets/schema.json ./userdocs/src/usage/schema.json
+userdocs/src/usage/schema.json: pkg/apis/eksctl.io/v1alpha5/assets/schema.json
+	cp $< $@
```

Comment:
What is this magic?

Reply:
`make` built-in variables! `$<` is the first dependency thingy and `$@` is the target.

Reply:
Can we have the actual names instead? I prefer it to be explicit than something I won't understand in one month from now :/